src/abstractdataframe/show.jl

```diff
@@ -689,26 +717,5 @@ julia> show(df, allcols=true)
 │ 3   │ 3     │ z      │
 ```
 """
-Base.show(io::IO,
-          df::AbstractDataFrame;
-          allrows::Bool = !get(io, :limit, false),
-          allcols::Bool = !get(io, :limit, false),
-          splitcols = get(io, :limit, false),
-          rowlabel::Symbol = :Row,
-          summary::Bool = true,
-          eltypes::Bool = true,
-          truncate::Int = 32) =
-    _show(io, df, allrows=allrows, allcols=allcols, splitcols=splitcols,
-          rowlabel=rowlabel, summary=summary, eltypes=eltypes, truncstring=truncate)
-
-Base.show(df::AbstractDataFrame;
-          allrows::Bool = !get(stdout, :limit, true),
-          allcols::Bool = !get(stdout, :limit, true),
```

Comment:
Note that we used different defaults when `io` is passed and when it's. This was the result of a long discussion at https://github.com/JuliaData/DataFrames.jl/pull/1506#discussion_r217939013 (see also other threads), because by default `stdout` doesn't set `:limit`. The behavior we wanted was that the default REPL method wouldn't show all rows, but it would when specifying `io` explicitly because that's most likely what people want when printing to a file/buffer. What does this PR do in that case?

Reply:
Good catch! I did not understand this specific behavior. I revert this part to keep the old behavior. Thanks!